styles.css

```diff
@@ -3,69 +3,186 @@
 }
 
 body {
-    font-family: 'Barlow', sans-serif;
-    padding: 2rem;
+    font-family: 'Rubik', sans-serif;
     color: #262729;
+    display: grid;
+    grid-gap: 10px;
+    grid-template-rows: 1fr auto 0.2fr;
+    margin: 0; 
+    background-color: #ecdfdb;
+    line-height: 1.5;
 }
 
 header {
-    background-color: moccasin;
+    background-color: #3b27ba;
+    text-align: center;
+    padding-bottom: 1.5rem;
+    color: #13ca91;
 }
 
+header a {
+    text-decoration: none;
+    color: #13ca91;
+  }
+
 .nav {
-    color: 
+    padding: 1rem;
 }
 
-
 h1 {
-    color: indianred;
+    font-style: bold;
+    font-weight: 800;
 }
 
 h2 {
     text-decoration: underline;
+    color: #3b27ba;
 }
 
-
-table {
-    background-color: gainsboro;
-    border: solid black 1px; 
-    width: 800px;
-    font-weight: 100;
+h3 {
+    color: #3b27ba;
 }
 
-th {
-    text-align: left;
-    padding: 1rem 1rem;
+.linkcontainer {
+    display: flex;
+    justify-content: space-around;
+    border: solid 0.1rem #13ca91;
 }
 
-th.date {
-    width: 10%;
+.linkcontainer_item {
+    padding: 1rem;
+    text-align: center;
 }
 
-th.job {
-    width: 30%
+.hobbies, .skills {
+    text-align: center;
+    border: solid 0.1rem #13ca91;
 }
 
-th.company {
-    width: 50%;
+ul {
+    padding: 0;
 }
 
-.linkcontainer {
-    display: flex;
-    justify-content: center;
+li {
+    display: inline-block;  
+    background-color: #13ca91;
+    padding: 0.4rem;  
 }
 
-.linkcontainer_item {
-    border: solid 1px black;
+.aboutme {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 0.05fr 0.5fr auto 0.3fr;
```

Comment:
Full numbered fractions might make it easier to understand the grid:
`grid-template-rows: 5fr 5fr auto 3fr`